include/cupla/types.hpp

```diff
@@ -50,12 +50,18 @@
 #   define ALPAKA_ACC_CPU_B_SEQ_T_SEQ_ENABLED 1
 #endif
 
+#ifdef ALPAKA_ACC_CPU_B_TBB_T_SEQ_ENABLE
```

Comment:
just a note: that is a bit weird, if it is defined a `0` one might want to keep that `0` (in all options)

Reply:
In alpaka it is only defined without any value, but I will look over this part soon again and will check if I can remove the precompiler constructs.